homeassistant/components/saj/sensor.py

```diff
@@ -67,11 +69,14 @@
     # Use all sensors by default
     hass_sensors = []
 
+    kwargs = {}
+    if config.get(CONF_NAME):
+        kwargs["inverter_name"] = config[CONF_NAME]
+
     for sensor in sensor_def:
-        hass_sensors.append(SAJsensor(sensor))
+        hass_sensors.append(SAJsensor(sensor, **kwargs))
```

Comment:
Since this is the only optional parameter, this sequence can probably be simplified:

```python
for sensor in sensor_def:
    hass_sensors.append(SAJsensor(sensor, inverter_name=config.get(CONF_NAME))
```

Reply:
Indeed, thanks!